Description: CVE-2025-13151 - Fix buffer overflow in libtasn1
 Increased buffer size from 2*ASN1_MAX_NAME_SIZE+1 to 2*ASN1_MAX_NAME_SIZE+2
 to prevent one-byte stack overflow in asn1_expand_octet_string function.
 The overflow can occur when both definition name and element name are
 ASN1_MAX_NAME_SIZE characters long.
Origin: upstream, commit:d276cc495a2a32b182c3c39851f1ba58f2d9f9b8
Author: Vijay Sarvepalli
Last-Update: 2026-01-12

--- a/lib/decoding.c
+++ b/lib/decoding.c
@@ -1968,7 +1968,7 @@ int
 asn1_expand_octet_string (asn1_node_const definitions, asn1_node * element,
 			  const char *octetName, const char *objectName)
 {
-  char name[2 * ASN1_MAX_NAME_SIZE + 1], value[ASN1_MAX_NAME_SIZE];
+  char name[2 * ASN1_MAX_NAME_SIZE + 2], value[ASN1_MAX_NAME_SIZE];
   int retCode = ASN1_SUCCESS, result;
   int len, len2, len3;
   asn1_node_const p2;
